net/openfortivpn/files/openfortivpn.sh

```diff
@@ -138,6 +140,11 @@ proto_openfortivpn_setup() {
 	[ -f /etc/openfortivpn/user-key-$config.pem ] && append_args "--user-key=/etc/openfortivpn/user-key-$config.pem"
 	[ -f /etc/openfortivpn/ca-$config.pem ] && append_args "--ca-file=/etc/openfortivpn/ca-$config.pem"
 
+	[ "$saml_login" = 1 ]  && {
+		[ -n "$saml_login_port" ] && append_args "--saml-login=$saml_login_port"
+		[ ! -n "$saml_login_port" ] && append_args "--saml-login"
```

Comment:
```suggestion
		if [ -n "$saml_login_port" ]; then
			append_args "--saml-login=$saml_login_port"
		else
			append_args "--saml-login"
		fi
```

please use if-else block here, otherwise LGTM

Reply:
Done

Reply:
just found a simpler syntax:
`[ "$saml_login" = 1 ] && append_args "--saml-login${saml_login_port:+=$saml_login_port}"`

if-else block is more clear, both are ok for me.